docs/pages/database-access/guides/rds.mdx

```diff
@@ -294,9 +391,12 @@ $ teleport start --config=/etc/teleport.yaml
 ```
 
 The agent will discover all RDS instances and Aurora clusters according to the
-configuration and register them in the cluster. In addition to the primary
-endpoints of the discovered Aurora clusters, their reader and custom endpoints
-will also be registered.
+configuration and register them in the cluster, when database type `rds` is
```

Comment:
Let's remove "when database type `rds` is configured" from the end of this sentence and add something more precise to the beginning of the sentence instead, e.g., "When `db_service.aws[*].types` includes `rds`, ...".